layouts/partials/downloadButton.html

```diff
@@ -1,6 +1,6 @@
-<a href="{{ "docs/quick-start/minimal-kubesphere-on-k8s/" | relLangURL }}" target="_blank" rel="noopener noreferrer">
+<a href="{{ "docs/quick-start/minimal-kubesphere-on-k8s/" | relLangURL }}" aria-label="Install on Kubernetes" target="_blank" rel="noopener noreferrer">
   <button class='download-btn download-btn1 common-btn'>{{ i18n "Install on Kubernetes" }}</button>
 </a>
-<a href="{{ "docs/quick-start/all-in-one-on-linux/" | relLangURL }}" target="_blank" rel="noopener noreferrer">
+<a href="{{ "docs/quick-start/all-in-one-on-linux/" | relLangURL }}" aria-label="Install on Linux" target="_blank" rel="noopener noreferrer">
```

Comment:
I think we‘d better use i18n to transform it.

Reply:
@Arhell PTAL, thanks.

Reply:
done